usr/lib/pkcs11/ep11_stdll/ep11_specific.c

```diff
@@ -2380,15 +2408,19 @@ static CK_RV import_DSA_key(STDLL_TokData_t * tokdata, SESSION * sess,
             TRACE_INFO("%s wrapping unwrap key rc=0x%lx blob_size=0x%zx\n",
                        __func__, rc, *blob_size);
         }
+
+        cleanse_attribute(dsa_key_obj->template, CKA_VALUE);
     }
 
 import_DSA_key_end:
```

Comment:
check_key_attributes exit point should be mapped to this label ?

Reply:
fixed